### PR TITLE
Amend gateway documentation

### PR DESCRIPTION
**Pull request by fgiloux**
Amend gateway documentation to refer to kcp GLBC (instead of the PoC implementation)

Signed-off-by: Frederic Giloux <fgiloux@redhat.com>

### CODE REVIEW

File: docs/gateway.md [hunk 1]
```diff
@@ -10,20 +10,26 @@ The approach taken for addressing it is to introduce a gateway (an haproxy conta
 
 ![Gateway architecture](./images/haproxy.png)
 
-- GLB is kcp global load balancer. This is a work in progress. Currently a development version is based on an ingress-controller that automatically configures an envoy proxy.
+- GLB is [kcp global load balancer](https://github.com/Kuadrant/kcp-glbc). This is a work in progress. The current version automatically configures DNS, e.g. route 53, so that the hostname points to the ingress endpoint of the cluster where the matching workload (Service, Deployment) have been deployed. It also integrates with cert-manager to configure a certificate that matches the hostname.
```
Comment (Roming22) [inline docs/gateway.md:13]: `matching workload (...) has` or `matching workloads (...) have`

File: docs/gateway.md [hunk 2]
```diff
@@ -10,20 +10,26 @@ The approach taken for addressing it is to introduce a gateway (an haproxy conta
...
+3. The ingress router forwards the call to its backend the gateway (HAProxy container).
+4. HAProxy uses path-based routing to forward `/trigger` requests to the Pipelines as Code Service.
+5. The Service, implemented through iptables rules or a cloud provider's load balancer most of the time, forwards the packets to the trigger pod.
 6. The trigger pod processes the request.
 
 ## Installation
 
+### Prerequisites
+
+[kcp GLBC](https://github.com/Kuadrant/kcp-glbc) needs to have been deployed. [Instructions](https://github.com/Kuadrant/kcp-glbc/blob/main/docs/deployment.md) are provided in its GitHub repository.
```
Comment (Roming22) [inline docs/gateway.md:29]: `must be deployed`?

File: docs/gateway.md (hunk 1 above)
Comment (fgiloux) [reply docs/gateway.md:13]: Thanks. Fixed.

File: docs/gateway.md (hunk 2 above)
Comment (fgiloux) [reply docs/gateway.md:29]: Thanks. Improved.

**Review by Roming22 — APPROVED** [review]

**Pull request merged**